astro/src/content/docs/get-started/start-here/step-3.mdx

```diff
@@ -0,0 +1,78 @@
+---
+title: "Step 3 - Protected pages"
+description: Learn how to use FusionAuth by using an example application that has a protected page defined.
+section: get started
+subcategory: start here
+navOrder: 0
+codeRoot: https://raw.githubusercontent.com/fusionauth/fusionauth-example-get-started/main
+---
+
+<div className="border border-green-700 px-3 rounded-md">
+  <i className="fa-duotone fa-stopwatch fa-2x mr-2 text-green-700 align-middle"/> Estimated time to complete this step: **5 minutes**
+</div>
+
+## Protecting pages
+
+Now that we have the FusionAuth and the example app running, let's take a look at how the application protects pages that require a user to be logged in.
```

Comment:
Could we make this more concrete and specific to this example? i.e. tell the reader we are going to show them how the Changebank app only allows a logged-in user to access the account page. This not only protects access to people's accounts, but it also tells the Changebank app whose account to load up. Or something like that.